blueprints/route/index.js

```diff
@@ -0,0 +1,147 @@
+/* eslint-env node */
```

Comment:
Is this file like _blueprints/component/index.js_, where its the same content as already exists?  If so, the same question I posed for _blueprints/component/index.js_ also applies to this one.